web/src/components/timeControls.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
```

Comment:
It's a bit weird how it first changes to "June 6 ...." for all ticks, until data is loaded. Dunno if it's easily fixable? :)
![Kapture 2022-06-07 at 14 07 05](https://user-images.githubusercontent.com/3296643/172375611-79151282-bed7-40cd-a0dd-6077274464b4.gif)

Reply:
The most robust solution would be to have some sort of loading UI I am not sure how it would look with a spinner. Perhaps there is some skeleton UI solution? 

cc @rasmusjeppsen